contribute/breaking-changes-guide/breaking-changes-guide.md

```diff
@@ -32,36 +32,36 @@ Any change that causes dependent software to behave differently is considered to
 
 Levitate can list exported members of an NPM package or imports used by an NPM package, _but it is most commonly used for comparing different versions of the same package to see changes in the exported members._
 
-A Github workflow runs against every pull request and comments a hint in case there are
-possible breaking changes. It also adds the `"breaking change"` label to the pull request.
+A GitHub workflow runs against every pull request and comments a hint in case there are possible breaking changes.
+It also adds the `breaking change` label to the pull request.
 
 ## What does the CI workflow look like?
 
 <img src="./breaking-changes-workflow.png" alt="CI workflow" width="700" />
 
 ## What do comments on my PR mean?
 
-![Levitate comment](./breaking-changes-comment-screenshot.png)
+![Example Levitate comment on PR](./breaking-changes-comment-screenshot.png)
 
-Receiving a comment like the one above does not necessarily mean that you actually introduced breaking
-changes. This is because certain edge cases are still not covered by the tool, but there is a good chanceso we raise it to your attention.
+Receiving a comment like this does not necessarily mean that you actually introduced breaking
+changes. This is because certain edge cases are still not covered by the tool, but there is a good chance that they may happen, so we call it to your attention.
 
 By clicking the links in the comment ("more info" or "Check console output") you can view more detailed information about what triggered the notification.
 
 **Removed exported members** (console view):<br />
 This means that some previously exported members won't be available in the newer version of the package, so dependent plugins can break.
 
-![](./breaking-changes-console-screenshot-1.png)
+![Removed exported members](./breaking-changes-console-screenshot-1.png)
 
 **Changed an existing member** (console view):<br />
 This means that a member was changed in a way that can break dependent plugins.
 
-![](./breaking-changes-console-screenshot-2.png)
+![Changed an existing member](./breaking-changes-console-screenshot-2.png)
 
 **No breaking changes** (console view):<br />
 Seeing this suggests that while changes were made, most probably none of them were breaking. You are good to go! 👏
 
-![](./breaking-changes-console-screenshot-3.png)
+![No breaking changes](./breaking-changes-console-screenshot-3.png)
```

Comment:
Thanks for adding alt text. I think they could all be more descriptive if we truly want to be sympathetic to screen readers. https://grafana.com/docs/writers-toolkit/write/image-guidelines/#alt-text has some good guidelines.

Reply:
made more descriptive.